polynote-kernel/src/main/scala/polynote/config/PolynoteConfig.scala

```diff
@@ -291,15 +300,30 @@ object PolynoteConfig {
           .as(Json.fromJsonObject(JsonObject.empty))
     }
 
-    val configIO = for {
-      configJson  <- parsed
-      defaultJson <- default
-      merged = defaultJson.deepMerge(configJson) // priority goes to configJson
-      parsedConfig <- ZIO.fromEither(merged.as[PolynoteConfig])
-      _ <- ZIO.when(parsedConfig.behavior.kernelIsolation == KernelIsolation.Never) {
-        Logging.warn("Configuration value `behavior.kernel_isolation: never` is deprecated and will be removed")
-      }
-    } yield parsedConfig
+    val configIO = parsed
```

Comment:
is this desugared for debugging purposes?

Reply:
Oops, yes. I [put it back](https://github.com/polynote/polynote/pull/1432/commits/670449ed7a5a6f360a85d211480d3e1b9503c1d4#diff-95c5c982fc4bf39d8e0d97de3fdb1ed4601fd8c78723869a676fb38a2f41f728R304-R312)!